packages/core/src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -804,6 +859,50 @@
                 {
                     "type": "codewhispererCustomizationArn",
                     "required": false
+                },
+                {
+                    "type": "cwsprChatHasContextList",
```

Comment:
non-blocker: Are there plans to move this to the common repo? IIRC this metric was initially created here to allow you guys to move fast.

Reply:
Yeah if you take over that would be great. We have plans to consume this metric across all IDEs for some separate work.